custom-elements/reactions/Document.html

```diff
@@ -61,6 +61,7 @@
     assert_array_equals(element.takeLog().types(), ['constructed', 'connected']);
 
     container.focus();
+    contentDocument.getSelection().collapse(container, 1);
     contentDocument.execCommand('delete', false, null);
```

Comment:
I also tried to change `delete` to `forwarddelete`, but Firefox does not work well due to [bug 1850666](https://bugzilla.mozilla.org/show_bug.cgi?id=1850666) which may not be fixed so quickly.